pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -97,9 +103,30 @@
   social_maker_title: "I'm a Maker!"
   social_maker_desc: "Code.org physical computing and maker resources for grades 6-12."
 
+  social_blockchain_title: "How Blockchain Works"
+  social_blockchain_desc: "Videos and lessons that explore what blockchain is and why it's important."
+
   pl_superhero_title: "Help students become superheroes!"
   pl_superhero_desc: "Join our highly supportive Professional Learning Program for middle and high school educators."
 
+  blockchain_hero_heading: 'How Blockchain Works'
+  blockchain_hero_desc: 'Dive into the world of blockchain with "How Blockchain Works," our new video series and accompanying lessons!'
+  blockchain_intro_heading: 'See how blockchain works'
+  blockchain_intro_desc: 'The video series, made in partnership with Coinbase, features industry experts and aims to demystify this technology.'
+  blockchain_intro_list_heading: 'This series will:'
+  blockchain_intro_list_item_01: 'Explain what blockchain is'
+  blockchain_intro_list_item_02: "Explore why it's important"
+  blockchain_intro_list_item_03: 'Unpack its societal impacts — both good and bad'
+  blockchain_lessons_heading: 'Blockchain lessons for your class'
+  blockchain_lessons_desc: 'Bring blockchain to life in your classroom with lessons that correspond with the above videos! Each lesson can stand alone or as part of a "How Blockchain Works" series.'
```

Comment:
@castro-jorge lines 120 `blockchain_lessons_heading` and 121 `blockchain_lessons_desc` are the strings that need to have the "(English only)" blurb applied when translated.

Reply:
I'm curious how this will happen –– how will the english version *not* show "(English only)" but the other languages will? I thought we needed to add a check for that in the code, but could be off